### ISSUE TITLE
Custom Status Line format?Where is it?

### ISSUE THREAD

**Issue by DYFeng** [fholgado/minibufexpl.vim#50]
It seems that MBE uses it's own custom Status Line format to reduce the unwanted information,but I can't find anything about it in the source.
add this line in the source
'''vim
set statusline=%F
'''

**Comment by juanolon** [comment]
hi, i found this solution, hope it helps..
"http://dev.gentoo.org/~bass/configs/vimrc.html
if has("autocmd")
    au FileType qf
                \ if &buftype == "quickfix" |
                \     setlocal statusline=%2_%-3.3n%0_ |
                \     setlocal statusline+=\ [Compiler\ Messages] |
                \     setlocal statusline+=%=%2*\ %<%P |
                \ endif

```
fun! FixMiniBufExplorerTitle()
    if "-MiniBufExplorer-" == bufname("%")
        setlocal statusline=%2*%-3.3n%0*
        setlocal statusline+=\[Buffers\]
        setlocal statusline+=%=%2*\ %<%P
    endif
endfun

au BufWinEnter *
            \ let oldwinnr=winnr() |
            \ windo call FixMiniBufExplorerTitle() |
            \ exec oldwinnr . " wincmd w"
```

endif

**Comment by DYFeng** [comment]
THX very much.It works

**Comment by weynhamz** [comment]
Hey, this should be fixed in techlivezheng/vim-plugin-minibufexpl@a76acea and techlivezheng/vim-plugin-minibufexpl@9410d5c. Also, techlivezheng/vim-plugin-minibufexpl@7584a7a and techlivezheng/vim-plugin-minibufexpl@832ba52 should make MBE work fine with Powerline. After techlivezheng/vim-plugin-minibufexpl@1aeb012, you can use 'g:miniBufExplStatusLineText' to specify the status line for MBE. All these fixes are currently in [the develop branch of my fork](https://github.com/techlivezheng/vim-plugin-minibufexpl/tree/develop), but soon it will be available in @fholgado's repo with coming 6.5 release.
